content/sensu-go/6.1/plugins/supported-integrations/_index.md

```diff
@@ -0,0 +1,73 @@
+---
+title: "Supported Integrations"
+description: "Start using Sensu plugins with these supported integrations. Sensu integrations are available for PagerDuty, Ansible, Chef, Jira, Elasticsearch, InfluxDB, and more."
+product: "Sensu Go"
+version: "6.1"
+weight: 80
+layout: "single"
+toc: true
+menu:
+  sensu-go-6.1:
+    parent: plugins
+    identifier: supported-integrations
+---
+
+Sensu integrations include plugins, libraries, and runtimes that extend Sensu's functionality and allow you to automate your monitoring and observability workflows.
+You can also rely on Sensu's integrations to get work done with Sensu as part of your existing workflows.
+
+Integrations are service-specific and have different setup and configuration requirements.
+Each integration has self-contained documentation with in-depth information about how to install and use it.
+Many of the supported integrations include curated quick-start templates that you only need to edit to match your configuration.
+
+Although this category focuses on our most popular supported integrations, you can find more supported-, Enterprise-, and community-tier integrations at [Bonsai, the Sensu asset index][1].
+
+## Alerting and incident management
+
+- [EC2][2]
+- [Email][3]
+- [Jira][4]
+- [PagerDuty][5]
+- [ServiceNow][6]
+- [Slack][7]
+
+## Auto-remediation
+
+- [Ansible][8]
+- [Rundeck][9]
+- [SaltStack][10]
+
+## Deregistration
+
+- [Chef][11]
+- [Puppet][12]
+
+## Status and metrics processing and storage
```

Comment:
```suggestion
## Time-series and long-term event storage
```

Consider renaming this section? Time-series or TSDB is a pretty common term which folks might grok more immediately.